### PR TITLE
Troubleshooting for error certificate signed by unknown authority

### PR DESCRIPTION
**Pull request by DonSchenck**
NOTE WELL: The "topic" is 'Chair by Rackspace'. This is intentional *until we have a name* for this service.

DO NOT MERGE UNTIL THE TOPIC IS FIXED.

### CODE REVIEW

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Error 'Certificate signed by unknown authority'
+permalink: /docs/references/troubleshooting-certificate-signed-by-unknown-authority/
+topics:
+  - docker
+  - troubleshooting
+  - Chair by Rackspace
```
Comment (rgbkrk) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Name is now "Carina by Rackspace"

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+When running a docker command, as shown in the following example, you may receive an error similar to the following:
```
Comment (rgbkrk) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: You can trim this down to just:

> If you see this error with `docker`:

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```bash
+$ docker ps -a
+An error occurred trying to connect: Get https://256.130.0.42:2376/v1.20/containers/json?all=1: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "serial:11002205957147154644")
+```
+
+This error occurs when the credentials you are using on your local machine do not match the credentials expected by the cluster.
```
Comment (rgbkrk) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Then the credentials on your local machine do not match the credentials expected by the cluster.

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Resolution
+Download and use the credentials generated by and assigned to the cluster in question.
```
Comment (rgbkrk) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Link to the UI or show how to do it with [`carina`](https://github.com/rackerlabs/carina/blob/master/README.md#carina-client).

**Comment by rgbkrk** [comment]
How did you manage to get this error to come up when using Carina? Undefining some environment variables, only setting some?

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+title: Error 'Certificate signed by unknown authority'
```
Comment (kmsholcomb) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Change the title to: Troubleshooting a certificate error

Comment (kmsholcomb) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Under `title`, add these fields:

author: Don Schenck Last <don.schenck@rackspace.com>
date: 2015-10-22

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+permalink: /docs/references/troubleshooting-certificate-signed-by-unknown-authority/
+topics:
```
Comment (kmsholcomb) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Add a description field:

description: Troubleshoot the error "Certificate signed by unknown authority"

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md (hunk 2 above)
Comment (kmsholcomb) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: I would suggest leaving it a bit wordy:

When you run a `docker` command, as shown in the following example, you might see the following error message:

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md (hunk 3 above)
Comment (kmsholcomb) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: Again, a bit wordier:

This error occurs when the credentials that you are using on your local machine do not match the credentials expected by the cluster.

File: _references/troubleshooting-certificate-signed-by-unknown-authority.md (hunk 4 above)
Comment (kmsholcomb) [inline _references/troubleshooting-certificate-signed-by-unknown-authority.md]: I think that you can delete "in question."

I would suggest providing a link to the "Download Carina credentials" article:

For instructions, see [Download Carina credentials](/docs/references/rcs-credentials/).

**Comment by kmsholcomb** [comment]
Hi @DonSchenck: I added my comment to Kyle's in the file accessible from this page. If I had a different suggestion than Kyle, I noted that. If you implement my changes, you can consider this a :+1:. Please let me know if you have any questions. Thanks!

(FYI: When I tried to View the file, I got a 404 error. When they make the repo public, it broke the connection between the repo and all the forked repos. I saw similar error message when that happened, so that could be the cause. If you didn't refork yet, I don't know if your PR is still connected to the file on your fork. Of course, I could be wrong, but if things don't seem connected like they should be, that might be the reason.)

**Comment by everett-toews** [comment]
@DonSchenck Is this still valid?

Should it perhaps be closed and maybe resubmitted later?

**Comment by DonSchenck** [comment]
The only way I know to cause this error is to delete a cluster and then create a new one with the same name, but fail to download the new credentials.

Given how unlikely that scenario is, this can be tabled.

**Comment by rgbkrk** [comment]
I've definitely run into this with repeated use of the `carina` CLI, which I'm looking to address with https://github.com/getcarina/carina/issues/63.
